cmd/ctr/commands/run/run_windows.go

```diff
@@ -118,7 +88,8 @@ func newContainer(ctx gocontext.Context, client *containerd.Client, context *cli
 	)
 }
 
-func newTask(ctx gocontext.Context, client *containerd.Client, container containerd.Container, _ string, tty, nullIO bool) (containerd.Task, error) {
+// NewTask creates a new task
+func NewTask(ctx gocontext.Context, client *containerd.Client, container containerd.Container, _ string, tty, nullIO bool) (containerd.Task, error) {
```

Comment:
Is this getting duplicated with `tasks.NewTask`?

Reply:
yes, thanks for the catch!